test/bundle.test.ts

```diff
@@ -32,7 +32,7 @@ describe.skipIf(process.env.SKIP_BUNDLE_SIZE === 'true' || process.env.ECOSYSTEM
     const serverDir = join(rootDir, '.output/server')
 
     const serverStats = await analyzeSizes(['**/*.mjs', '!node_modules'], serverDir)
-    expect.soft(roundToKilobytes(serverStats.totalBytes)).toMatchInlineSnapshot('"64.6k"')
+    expect.soft(roundToKilobytes(serverStats.totalBytes)).toMatchInlineSnapshot('"64.7k"')
```

Comment:
@pi0 We can directly use `#internal/nitro` but it does have a bundle size cost.

Reply:
No worries i think we can check possible tree-shaking issues later. Since we were already using `/app`, it should be fine to keep because anyway i cannot break in a non major version of nitro 🙈 (https://github.com/nuxt/nuxt/blob/main/packages/nuxt/src/core/runtime/nitro/renderer.ts)